assets/dev/js/editor/controls/media.js

```diff
@@ -24,15 +24,30 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		} );
 	},
 
+	/**
+	 * Get library type for `wp.media` using a given media type.
+	 *
+	 * @param {String} mediaType - The media type to get the library for.
+	 * @returns {String}
+	 */
+	getLibraryType: function( mediaType ) {
+		if ( mediaType ) {
+			return ( 'svg' === mediaType ) ? 'image/svg+xml' : mediaType;
+		}
+
+		return this.getMediaType();
```

Comment:
When no argument is being passed to this function, the value that being returned from the this.getMediaType() should still be "image/svg+xml" when the mediaType is SVG.

So it should be something like:
```suggestion
		if ( ! mediaType ) {
			mediaType = this.getMediaType();
		}

		return ( 'svg' === mediaType ) ? 'image/svg+xml' : mediaType;
```